front/pages/w/[wId]/data-sources/vaults/[vaultId]/categories/[category]/data_source_views/[dataSourceViewId].tsx

```diff
@@ -0,0 +1,81 @@
+import type { DataSourceViewType, UserType, VaultType } from "@dust-tt/types";
```

Comment:
Can we stuck with [dsvId] out of consistency across the entire codebase 🙏

Same for the other endpoint?